controller/search_blackbox_test.go

```diff
@@ -575,6 +577,19 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 
 	wirepo := workitem.NewWorkItemRepository(s.DB)
 
+	labelNames := []string{"IMPORTANT", "backend", "UI", "REST"}
+	testFixtures, err := tf.NewFixture(s.DB, tf.Labels(4, func(f *tf.TestFixture, idx int) error {
+		f.Labels[idx].Name = labelNames[idx]
+		return nil
+	}))
+	require.Nil(s.T(), err)
+	require.Len(s.T(), testFixtures.Labels, 4) // not required but being explicit
```

Comment:
No need to verify this. Each recipe function of the test fixture adds a [check](https://github.com/fabric8-services/fabric8-wit/blob/6ce3dd28ee682bd5f58350157cd13280ba8e2f53/test/testfixture/testfixture.go#L120) for the number of created entities (see [here](https://github.com/fabric8-services/fabric8-wit/blob/6ce3dd28ee682bd5f58350157cd13280ba8e2f53/test/testfixture/recipe_funcs.go#L427) for labels) and that test will be [executed before `NewFixture` returns](https://github.com/fabric8-services/fabric8-wit/blob/6ce3dd28ee682bd5f58350157cd13280ba8e2f53/test/testfixture/testfixture.go#L227). So if the length really is different to `4`, then the `require.Nil(s.T(), err)` line above will abort the test before this line can be reached.

Reply:
was aware of that hence added that comment next to the line :P
Now removed in https://github.com/fabric8-services/fabric8-wit/pull/1640/commits/46a3fe097d1ef83dfab077182e2ed21aee75c515